met-api/src/met_api/resources/contact.py

```diff
@@ -88,3 +88,26 @@ def get():
             return ActionResult.success(result=widgets)
         except (KeyError, ValueError) as err:
             return ActionResult.error(str(err))
+
+
+@cors_preflight('PUT')
+@API.route('/')
+class Contact(Resource):
+    """Contact controller class."""
+
+    @staticmethod
+    # @TRACER.trace()
+    @cross_origin(origins=allowedorigins())
+    @auth.require
+    def put():
+        """Update Contact."""
+        try:
+            contact_data = TokenInfo.get_contact_data()
```

Comment:
TokenInfo class is used to retrieve information about the logged in user, for your case you should use user_id = TokenInfo.get_id() to get the user id that is updating the contact, and pass the user_id along to you service class, there you'll need to update the updated_by in the contact record

Reply:
awesome did not know this :)

Reply:
Is this resolved @tiagograf-AOT ?